packages/paste-website/src/pages/getting-started/engineering.mdx

```diff
@@ -26,7 +36,11 @@ export const pageQuery = graphql`
 
 ## Installation
 
-Currently, every package is standalone in the paste project.  While this means you only get what you need, it makes it a little more tedious to get started.  We hope to address this in the near future, but for now you can run:
+The best way to use Paste is by using the core package. Paste Core provides all of the current Paste components which are ready for use.
+
+Paste Core requires external dependencies, and other Paste system packages in order to function properly.
+
+Install the following dependencies using yarn:
 
 #### External dependencies
```

Comment:
I know this might feel redundant, but I might like to see these lists as real lists above the bash snippet. That way it's a bit easier for consumers to scan what they are.

For the externals you can then list the version numbers too.

Reply:
That goes for both external and system packages

Reply:
I added package and version tables above each yarn command.